llvm-bazel/llvm-project-overlay/mlir/BUILD

```diff
@@ -955,10 +955,10 @@ cc_library(
 filegroup(
     name = "ShapeOpsTdFiles",
     srcs = [
-        ":StdOpsTdFiles",
         "include/mlir/Dialect/Shape/IR/ShapeBase.td",
         "include/mlir/Dialect/Shape/IR/ShapeOps.td",
         "include/mlir/Interfaces/InferTypeOpInterface.td",
+        ":StdOpsTdFiles",
```

Comment:
This is unrelated and snuck in via buildifier. Sorry! I can revert this bit if desired

Reply:
Nah, it's fine.